src/Nova/Post.php

```diff
@@ -52,7 +61,6 @@ public function fields(Request $request)
             Slug::make('Slug')->from('Title'),
             nova('layout') ? BelongsTo::make('Layout', 'layout', nova('layout'))->nullable() : null,
             DateTime::make('Published', 'published_at'),
-            nova('layout') ? BelongsTo::make('Layout', 'layout', nova('layout'))->nullable() : null,
```

Comment:
This change is unrelated to the topic of the PR, but I noticed the layout field was repeated twice in the resource.

Reply:
Nice catch.